WordPress/WordPressTest/SiteCreation/SiteDesignTests.swift

```diff
@@ -73,4 +73,42 @@ class SiteDesignTests: XCTestCase {
         let currentTitle = siteDesignPreviewVC.primaryActionButton.currentTitle
         XCTAssertEqual(expectedPrimaryTitle, currentTitle)
     }
+
+    /// Tests that the preview device on the Design view cannot be changed
+    func testSiteDesignPreviewDeviceIsAlwaysMobile() throws {
+
+        // given
+        let siteDesignVC = SiteDesignContentCollectionViewController(createsSite: false) { _ in }
+        let expectedDevice = PreviewDeviceSelectionViewController.PreviewDevice.mobile
+
+        // when
+        XCTAssertEqual(siteDesignVC.selectedPreviewDevice, expectedDevice)
+        siteDesignVC.selectedPreviewDevice = PreviewDeviceSelectionViewController.PreviewDevice.tablet
+
+        // then
+        XCTAssertEqual(siteDesignVC.selectedPreviewDevice, expectedDevice)
+    }
+
+    /// Tests that the preview device on the Preview view can be changed
+    func testSiteDesignPreviewPreviewDeviceCanBeChanged() throws {
```

Comment:
This comment makes it clear that the `PreviewPreview` in the function name is intentional :smile: .

Reply:
I could suggest `PreviewPreview` → `PreviewPreviewView` 😅

Reply:
I cringed a little when I made `testSiteDesignPreviewPreviewDeviceCanBeChanged` so I'll tweak that to a better name. 😄

Reply:
I ended up removing this test altogether after the last batch of changes as it seemed to lose its value.